---
layout: cv
title: Qike's CV
---
# Qike Li
Statistical Informatics PhD and data scientist

<div id="webaddress">


<i class="fa fa-user"></i> <a href="https://qikeli.github.io">Website</a>
|
<i class="fa fa-envelope"></i> <a href="mailto:qikeli@gmail.com">qikeli at gmail dot com</a>
|
<i class="fa fa-github"></i> <a href="http://github.com/QikeLi">QikeLi</a>
|
<i class="fa fa-twitter"></i> <a href="http://twitter.com/Qike_Li/">Qike_Li</a>
|
<i class="fa fa-file-pdf-o"></i> <a href="https://qikeli.github.io/files/CV-Qike-Li-2017.pdf">CV</a>
</div>


## Currently

I'm a Ph.D candidate at [Statistics GIDP](http://stat.arizona.edu/), University of Arizona. 

## Education

`2012-present`
__PhD Candidate in Statistics–Statistical Informatics track__  University of Arizona  

Statistics Graduate Interdisciplinary Program (GIDP),

Minor in Biostatistics

Advisors: Hao Helen Zhang (Statistics) & Yves A. Lussier (Bioinformatics)

Expected graduation date: August 2017

`2006-2009`
__MS Molecular Biology__ Guizhou University

`2001-2005`
__BS Biotechnology__ Zhengzhou University

## Appointment

`2014-present` 
__Research Assistant__ [Lussier Group, Center for Biomedical Informatics & Biostatistics](http://www.lussiergroup.org/),  University of Arizona

Developing statistical informatics methodology for precision medicine. Engaging in interdisciplinary research: working with an expert team of statisticians, physicians, engineers, biologists, geneticists, and computer scientists. Responsibilities include original research, statistical support, grant writing, and software engineering.

`2013-2014`
__Research Assistant__ Fionna McCarthy Group, University of Arizona

Developed statistical informatics methodology for RNA-Seq analysis.

`2010-2012`
__Research Technician__ Rick Michod Group,  University of Arizona

Studied the evolution of multicellularity by using bioinformatics, comparative genomics, phylogenetic analysis and molecular biology.g


## Publications<br /> [[Google Scholar profile]](https://scholar.google.com/citations?user=aB0lJKkAAAAJ&hl=en)

<!-- ### Journals -->

`2017` Gardeux, Vincent, Joanne Berghout, Ikbel Achour, A. Grant Schissler, **Qike Li**, Colleen Kenost, Jianrong Li, Yuan Shang, Anthony Bosco, Saner Don, Marilyn J. Halonen, Daniel J. Jackson, Haiquan Li, Fernando D. Martinez, Yves A. Lussier. "A genome-by-environment interaction classifier for precision medicine: personal transcriptome response to rhinovirus identifies children prone to asthma exacerbations" *Journal of the American Medical Informatics Association*  *(in press)*

`2017` **Li, Qike**, A. Grant Schissler, Vincent Gardeux, Ikbel Achour, Colleen Kenost, Joanne Berghout, Haiquan Li, Hao Helen Zhang, and Yves A. Lussier. "N-of-1-pathways MixEnrich: advancing precision medicine via single-subject analysis in discovering dynamic changes of transcriptomes." *BMC medical genomics* (2017) doi: DOI: 10.1186/s12920-017-0263-4 [[link]](https://bmcmedgenomics.biomedcentral.com/articles/10.1186/s12920-017-0263-4) [[code]](http://www.lussiergroup.org/publications/kMEn)

`2017` **Li, Qike**, A. Grant Schissler, Vincent Gardeux, Joanne Berghout, Ikbel Achour, Colleen Kenost, Haiquan Li, Hao Helen Zhang, and Yves A. Lussier. "kMEn: Analyzing noisy and bidirectional transcriptional pathway responses in single subjects." *Journal of biomedical informatics* (2017) doi: 10.1016/j.jbi.2016.12.009 [[link]](http://www.sciencedirect.com/science/article/pii/S1532046416301836)[[code]](http://www.lussiergroup.org/publications/MixEnrich)

`2016`  Schissler, A. Grant<sup>\*</sup>, **Qike Li<sup>\*</sup>**, James L. Chen, Colleen Kenost, Ikbel Achour, D. Dean Billheimer, Haiquan Li, Walter W. Piegorsch, and Yves A. Lussier. "Analysis of aggregated cell–cell statistical distances within pathways unveils therapeutic-resistance mechanisms in circulating tumor cells." *Bioinformatics* (2016) doi:https://doi.org/10.1093/bioinformatics/btw248 [[link]](https://academic.oup.com/bioinformatics/article/32/12/i80/2288654/Analysis-of-aggregated-cell-cell-statistical) (**\*** joint first authorship)

`2015`  Schissler, A. Grant<sup>\*</sup>, Vincent Gardeux<sup>\*</sup>, **Qike Li<sup>\*</sup>**, Ikbel Achour, Haiquan Li, Walter W. Piegorsch, and Yves A. Lussier. “Dynamic changes of RNA-sequencing expression for precision medicine: N-of-1-pathways Mahalanobis distance within pathways of single subjects predicts breast cancer survival.”  *Bioinformatics* (2015), doi:10.1093/bioinformatics/btv253 [[link]](https://academic.oup.com/bioinformatics/article-lookup/doi/10.1093/bioinformatics/btv253) (**\*** joint first authorship)

`2015`  Pookhao, Naruekamol, Michael B. Sohn, **Qike Li**, Isaac Jenkins, Ruofei Du, Hongmei Jiang, and Lingling An. “A two-stage statistical procedure for feature selection and comparison in functional analysis of metagenomes.” *Bioinformatics* (2015) doi:10.1093/bioinformatics/btu635 [[link]](https://doi.org/10.1093/bioinformatics/btu635)

`2015` Li, Haiquan, Nima Pouladi, Ikbel Achour, Vincent Gardeux, Jianrong Li, **Qike Li**, Hao Helen Zhang, Fernando D. Martinez, Joe G N Skip Garcia, Yves A. Lussier . “eQTL networks unveil enriched mRNA master
integrators downstream of complex disease-associated SNPs.” *Journal of Biomedical Informatics* (2015) doi:10.1016/j.jbi.2015.10.010 [[link]](https://doi.org/10.1016/j.jbi.2015.10.010)

`2015` Lussier, Yves a, Haiquan Li, Nima Pouladi, and **Qike Li**.“Accelerating precision biology and medicine with computational biology and bioinformatics.” *Genome Biology* (2014)  doi:10.1186/s13059-014-0450-y [[link]](https://doi.org/10.1186/s13059-014-0450-y)

`2015` Sohn, Michael B, Lingling An, Naruekamol Pookhao, and **Qike Li**. “Accurate genome relative abundance estimation for closely related species in a metagenomic sample.” *BMC Bioinformatics* (2014) doi:10.1186/1471-2105-15-242 [[link]](https://doi.org/10.1186/1471-2105-15-242).

## Presentations

### Short Course

`10/16/2016`
Short Course: Computational methods for precision medicine and single subject studies with genomes and transcriptomes. (_The 6th Annual Translational Bioinformatics Conference_, Jeju, Korea)

### Talks

`7/302017`
Joint Statistical Meetings (JSM) (_JSM_, Baltimore, USA)

`6/27/2017`
WNAR Annual Meeting (_WNAR_, Santa Fe, USA)

`10/16/2016`
Title ... (_The 6th Annual Translational Bioinformatics Conference_, Jeju, Korea)

## Technical skills

* R
* LaTeX
* UNIX and BASH
* Python
* HTML and CSS
* Adobe Illustrator
* Emacs and Elisp
* Cluster computing

## Links

* <i class="fa fa-user"></i> <a href="https://qikeli.github.io">Qike's Website</a>
* <i class="fa fa-envelope"></i> <a href="qikeli@gmail.com">qikeli at gmail dot com</a><br />
* <i class="fa fa-github"></i> <a href="http://github.com/QikeLi">QikeLi</a><br />
* <i class="fa fa-twitter"></i> <a href="http://twitter.com/Qike_Li">Qike_Li</a><br />
* <i class="fa fa-google"></i> <a href="http://scholar.google.com/citations?user=YMxsGpsAAAAJ">scholar</a>
* <i class="fa fa-file-pdf-o"></i> <a href="https://qikeli.github.io/files/CV-Qike-Li-2017.pdf">CV</a>

## References

Available on request.

<!-- ### Footer

Last updated: June 2017 -->
